React/finger-exercises/react/ecmascript/02 - promises/index.js

```diff
@@ -1,9 +1,20 @@
 // Hint: use setInterval, create a new Promise and measure time with Date.now()
 
-export function delay() {
-
+export function delay(time) {
+  const startTime = Date.now();
+  return new Promise((resolve, reject) => {
+    setInterval(function() {
```

Comment:
`setInterval` is not the ideal function to use here. You're trying to delay the execution of the function, not call it at intervals. You can find more information on `setInterval` here https://www.w3schools.com/jsref/met_win_setinterval.asp
Try to find a similar method that simply calls a function (`reject` or `resolve`) after an determined amount of time and doesn't execute the method again...

Reply:
The suggestion is wrong :(. Change it to `setTimeout` like @DLotero says